ci-operator/step-registry/workers-rhel/pre-hook/gcp/workers-rhel-pre-hook-gcp-commands.sh

```diff
@@ -20,11 +20,18 @@ cat > scaleup-pre-hook-gcp.yaml <<- 'EOF'
 - name: Configure RHEL machine on GCP
   hosts: new_workers
   any_errors_fatal: true
-  gather_facts: false
+  gather_facts: true
 
   tasks:
   - name: install checkpolicy
     yum: name=checkpolicy state=present
+
+  - name: List current enabled repos and also make repo metadata cache in advance
```

Comment:
@jianli-wei could you put this step before `install checkpolicy`, if the `repo metadata not available` error happens, `install checkpolicy` will fail.